zerver/models.py

```diff
@@ -2530,6 +2530,8 @@ class Stream(models.Model):
     }
     message_retention_days = models.IntegerField(null=True, default=None)
 
+    # False means we inherit user-level default
```

Comment:
Would be nice to add here that this will be used to set `push_notifications` field when a user is subscribing.